docs/using-the-jdbc-driver/using-plugins/UsingTheLimitlessConnectionPlugin.md

```diff
@@ -0,0 +1,42 @@
+# Using the Limitless Connection Plugin
+
+## What is Amazon Aurora Limitless Database?
+
+Amazon Aurora Limitless Database allows users to horizontally scale their Aurora Databases, handling millions of write transactions per second and managing petabytes of data. To learn more, please [read Amazon Aurora Limitless documentation](// TODO).
+
+## Why use the Limitless Connection Plugin?
+
+When connecting to an Amazon Aurora Limitless Database, clients will connect to the databases shard group endpoint, and be routed to a transaction router via Route 53.
```

Comment:
I would add a sentence here before.

"Limitless introduces a new endpoint for the databases - the shard group endpoint that is managed by Route 53."

Then...

"When using Limitless with the AWS JDBC Driver, users will now connect to this endpoint and be routed... etc."

Reply:
Done!